content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -140,6 +140,38 @@ Exit your shell:
 exit
 ```
 
+## Configure volume permission and ownership change policy for Pods
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+Kubernetes by default recursively changes ownership and permission of volume to match `fsGroup` specified in pod's
+`securityContext` when volume is mounted. Changing ownership and permissions of a large volume can be time consuming
+operation and that is where `fsGroupChangePolicy` field in `securityContext` comes in.
+
+**fsGroupChangePolicy** -  fsGroupChangePolicy defines behavior of changing ownership and permission of the volume
+before being exposed inside a Pod. This field only applies to volume types that support
+`fsGroup` controlled ownership (and permissions).
+This field has no effect on ephemeral volume types such as
+[`secret`](https://kubernetes.io/docs/concepts/storage/volumes/#secret),
+[`configMap`](https://kubernetes.io/docs/concepts/storage/volumes/#configmap),
+and [`emptydir`](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir). This field has two possible values:
```

Comment:
Optional change: the detail about ephemeral volumes could go inside a `{{< note >}}` shortcode.

Reply:
done